app/views/tracks/_overview.html.haml

```diff
@@ -4,7 +4,11 @@
       .pure-u-1-3
         .icon
           =image "track-page-mentors.png", "Track mentors"
-        %h3 #{number_with_delimiter @track.mentors.size} Mentors
+        %h3 #{number_with_delimiter @track.mentors.size}
+        -if @track.mentors.size == 1
```

Comment:
This needs to be indented below the h3. Currently this gives html of `<h3>5</h3> mentors` rather than `<h3>5 mentors</h3>`. 

You can also use the [pluralize](https://api.rubyonrails.org/classes/String.html#method-i-pluralize) helper to make life easier here. Could you maybe change this to something like:

```haml
%h3 #{number_with_delimiter @track.mentors.size} #{"Mentor".pluralize(@track.mentors.size)}
```